api/types.go

```diff
@@ -261,6 +261,8 @@ type EmbedRequest struct {
 type EmbedResponse struct {
 	Model      string      `json:"model"`
 	Embeddings [][]float32 `json:"embeddings"`
+
+	Metrics
```

Comment:
This is ok, however to avoid confusion as not all fields are provided, do we want to list out the fields individually?